homeassistant/components/glances/const.py

```diff
@@ -40,164 +44,183 @@ class GlancesSensorEntityDescription(SensorEntityDescription):
         name_suffix="used percent",
         native_unit_of_measurement=PERCENTAGE,
         icon="mdi:harddisk",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="disk_use",
         type="fs",
         name_suffix="used",
         native_unit_of_measurement=DATA_GIBIBYTES,
         icon="mdi:harddisk",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="disk_free",
         type="fs",
         name_suffix="free",
         native_unit_of_measurement=DATA_GIBIBYTES,
         icon="mdi:harddisk",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="memory_use_percent",
         type="mem",
         name_suffix="RAM used percent",
         native_unit_of_measurement=PERCENTAGE,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="memory_use",
         type="mem",
         name_suffix="RAM used",
         native_unit_of_measurement=DATA_MEBIBYTES,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="memory_free",
         type="mem",
         name_suffix="RAM free",
         native_unit_of_measurement=DATA_MEBIBYTES,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="swap_use_percent",
         type="memswap",
         name_suffix="Swap used percent",
         native_unit_of_measurement=PERCENTAGE,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="swap_use",
         type="memswap",
         name_suffix="Swap used",
         native_unit_of_measurement=DATA_GIBIBYTES,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="swap_free",
         type="memswap",
         name_suffix="Swap free",
         native_unit_of_measurement=DATA_GIBIBYTES,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="processor_load",
         type="load",
         name_suffix="CPU load",
-        native_unit_of_measurement="15 min",
         icon=CPU_ICON,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="process_running",
         type="processcount",
         name_suffix="Running",
```

Comment:
I wish I could update the name here to a more meaningful "Processes Running" (same for some other sensors) but apparently this integration uses name in unique_id which would cause duplicate entities on HA update which is a pretty annoying breaking change. Not sure what to do about it...

Reply:
Update the entity register in setup_entry: https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/entity_registry.py#L996

Search codebase for the function name and you will get some examples.

Reply:
Thanks! That looks promising but I'll probably try to do that in a separate PR. This migration may be quite annoying to implement properly in this case...